### PR TITLE
Add :seed:run gradle task

### PR DESCRIPTION
**Pull request by alkum**
Add support for and document usage of :seed:run task.

### CODE REVIEW

File: docs/sample-run-configs.md [hunk 1]
```diff
@@ -103,15 +103,52 @@ Copy the `Alice_clear` run configuration, rename it to `Alice_i2p` and change:
...
 ```
+# Using default settings
+./gradlew seed:run
+```
+
+For example, to start two local seeds, `bisq2_seed1` and `bisq2_seed2`, reachable on all 3 networks:
+
+```
+# Seed 1
+./gradlew seed:run \
```
Comment (chimp1984) [inline docs/sample-run-configs.md:117]: Maybe its better to either limit supportedTransportTypes to clearnet as tor and i2p require local manual setup for the local seed addresses.
Once we use protobuf for serialisation we can start those seeds on a server so its easier to run but currently with java serialisation it carries a slight security risk.

Comment (chimp1984) [reply docs/sample-run-configs.md:117]: So I would recommend to remove 
```
-Dbisq.networkServiceConfig.supportedTransportTypes.0=TOR \
-Dbisq.networkServiceConfig.supportedTransportTypes.1=I2P \
```

Comment (chimp1984) [reply docs/sample-run-configs.md:117]: Actually not sure if typesafe works with `supportedTransportTypes.2=CLEAR` alone or if index need to be 0 in that case. Propably more clear with 0.

**Review by chimp1984 — CHANGES_REQUESTED** [review]

File: docs/sample-run-configs.md (hunk 1 above)
Comment (alkum) [reply docs/sample-run-configs.md:117]: Yes good point, I removed tor and i2p, kept only the args for clearnet.

**Pull request merged**